config/settings.yml

```diff
@@ -45,3 +45,7 @@
         :ems_refresh_worker_amazon_network: {}
         :ems_refresh_worker_amazon_ebs_storage: {}
         :ems_refresh_worker_amazon_s3: {}
+:blacklisted_event_names:
```

Comment:
The problem with root level keys is, that they will be merged into the main settings.yml - which is editable via the UI 

I'm trying to get https://github.com/ManageIQ/manageiq/pull/13942 in and then we could have them under e.g. the `:amazon:` key